.github/workflows/vm-alpine.yml

```diff
@@ -30,13 +34,11 @@ jobs:
         branch: ${{ matrix.osversion }}
         arch: ${{ matrix.platform }}
         packages: "build-base check check-dev pkgconf"
-    - name: Build
+    - name: Build and Test
       shell: alpine.sh {0}
       run: |
+        set -e
         echo "cc -O2 ${{ matrix.cflags }}" > conf-cc
         echo "cc -s" > conf-ld
         make -j 2 it man NROFF=${{ matrix.nroff }}
```

Comment:
Good catch! Then if `make man` fails, we are warned about it.

Reply:
Thanks to @lighttpd (I saw this in their [pr.yml](https://github.com/lighttpd/lighttpd1.4/blob/79d011de6f942b2607f4d012e0372fb966c14bcf/.github/workflows/pr.yml#L35)).